### ISSUE TITLE
Search icon not clickable in the UI

### ISSUE THREAD

**Issue by banker** [redis-developer/basic-caching-demo-java#4]
The search icon should be clickable in the UI. Right now, the input only response to an onEnter event.

**Comment by ajeetraina** [comment]
Yes, the issue is reproducible over Cloud. Interestingly, if I try it locally, it works.

**Comment by jphaugla** [comment]
trying this locally did not work for me.  As a workaround, I just use curl to test instead of the UI.   I also had to change the port to 5001 which I did not debug...

 curl localhost:5001/repos/jphaugla
{"username":"jphaugla","repos":"52","cached":false}                            
 curl localhost:5001/repos/jphaugla
{"username":"jphaugla","repos":"52","cached":true}
